tests/test_measurements.py

```diff
@@ -73,6 +76,7 @@ def test_create_invalid_name(self):
         with self.assertRaises(KeyError):
             Measurement(metric, value, metric_set=self.metric_set)
 
+    @unittest.skip('FIXME DM8477 this is failing')
```

Comment:
@parejkoj Yeah, I'm using Astropy 1.2.1 as provided by lsstsw. `UnitTypeError` does seem to be a brand new API. This is what the test gives me otherwise:

```
>       with self.assertRaises(u.UnitTypeError):
E       AttributeError: 'module' object has no attribute 'UnitTypeError'
```

Shall I continue skipping this test and we can re-evaluate the exception handling. I'm not sure we necessarily change our Astropy pre-req in term for this Epic to close.